slycot/analysis.py

```diff
@@ -475,6 +475,108 @@ def ab08nd(n,m,p,A,B,C,D,equil='N',tol=0,ldwork=None):
         raise e
     return out[:-1]
 
+def ab08nz(n, m, p, A, B, C, D, equil='N', tol=0., lzwork=None):
+    """ nu,rank,dinfz,nkror,nkrol,infz,kronr,kronl,Af,Bf = ab08nz(n,m,p,A,B,C,D,[equil,tol,lzwork])
+
+    To construct for a linear multivariable system described by a state-space
+    model (A,B,C,D) a regular pencil (Af - lambda*Bf ) which has the invariant
+    zeros of the system as generalized eigenvalues.
+    The routine also computes the orders of the infinite zeros and the
+    right and left Kronecker indices of the system (A,B,C,D).
+
+    Required arguments:
+        n : input int
+            The number of state variables.  n >= 0.
+        m : input int
+            The number of system inputs.  m >= 0.
+        p : input int
+            The number of system outputs.  p >= 0.
+        A : input rank-2 array('d') with bounds (n,n)
```

Comment:
"with bounds (n,n) is" not terminology familiar to me.  Maybe "with A.shape >= (n,n)" ?

Reply:
See #100. It comes directly from f2py and is used in many other existing docstrings in that file.